src/GuestConfiguration/GuestConfiguration/Common/GuestConfigurationCmdletBase.cs

```diff
@@ -271,13 +278,18 @@ private string GetPolicySetCategory(PolicySetDefinition initiativeDefinition)
         {
             var categoryMetadata = JObject.Parse(initiativeDefinition.Metadata.ToString());
             var categoryMetadataDictionary = categoryMetadata.ToObject<Dictionary<string, object>>();
-            var category = categoryMetadataDictionary["category"].ToString();
-            return category;
+            if (categoryMetadataDictionary.ContainsKey("category"))
```

Comment:
use constant for "category"

Reply:
Wont fix for now.
String literals have been used everywhere in the file.
I can take up a new task to fix the string constants everywhere in the cmdlets.

---
In reply to: [258726975](https://github.com/Azure/azure-powershell/pull/8574#discussion_r258726975) [](ancestors = 258726975)